lib/travis/api/serialize/serializer.rb

```diff
@@ -1,5 +1,20 @@
 require 'active_model_serializers'
 
+# This hideousness courtesy of http://stackoverflow.com/a/8339255
+module ActiveSupport::JSON::Encoding
+  class << self
+    def escape(string)
+      if string.respond_to?(:force_encoding)
```

Comment:
Is string ever anything but a string? If so I would suggest removing the if and doing string = string.to_s.encode( ... etc. if it doesn't respond to force_encode, I doubt it would respond to gsub below. However, is this if statement protecting against a specific error you saw? Are there ever ruby strings that do not respond to force_encode? Similar comment for the json if below. Json.to_s can replace %("#{json}")